packages/cli/package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@janus-idp/cli",
+  "name": "@janus-idp/backstage-cli",
```

Comment:
Why do you rename the package?

Reply:
Just a mistake (at some point, to test I had renamed it to `dfatwork-pkgs/backstage-cli` to push it to my personal NPMJS org). Let me change this back

Reply:
Fixed